scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -153,15 +137,21 @@ task IsVcfOnChromosome {
 
     mkdir output
     touch output/~{output_vcf_name}
-    CHR=$(cat chrom.txt)
-    if [ $CHR = ~{chromosome} ]; then
-      cp ~{input_vcf} output/~{output_vcf_name}
-      echo "Including ~{input_vcf} as it is on chromosome ~{chromosome}."
-    else
-      touch output/~{output_vcf_name}
-      echo "Skipping ~{input_vcf} as it is not on chromosome ~{chromosome}."
-    fi
-  }
+    VCF_CHR=$(cat chrom.txt)
+    chromosomes=( ~{sep=' ' chromosomes} )
+
+    for i in "${cchromosomes[@]}"
```

Comment:
`chromosomes` is misspelled but this isn't blowing up? I'm wondering if the nonstandard-for-GVS bash options at the top are working as intended.

Reply:
I renamed all the "contig"s to "chromosomes" and messed this up (after the test runs listed above -- but, I was going to run at least one more to catch stuff like this).  Good catch -- fixed!